nvflare/private/fed/client/client_engine.py

```diff
@@ -74,6 +79,9 @@ def do_validate(self, req: Message):
 
         return "validate process started."
 
+    def new_context(self) -> FLContext:
+        return self.fl_ctx_mgr.new_context()
+
```

Comment:
Why do we need this function here?

Reply:
This is fully related to this bug. But we need this change for the client_engine.new_context().